rdr_service/tools/tool_libs/participant_summary_data_dump.py

```diff
@@ -0,0 +1,99 @@
+from rdr_service.model.participant import Participant
+from rdr_service.services.system_utils import list_chunks
+
+from rdr_service.tools.tool_libs.tool_base import cli_run, ToolBase
+
+tool_cmd = 'ps-data-dump'
+tool_desc = 'bulk data dump of participants from the PS API for vibrent'
+bucket_name = 'ptsc-metrics-all-of-us-rdr-prod'
+client = storage.Client()
+
+
+def upload_to_gcs(df, filename):
+    bucket = client.get_bucket(bucket_name)
+    filename = f'ops_data_api/{filename}'
+    blob = bucket.blob(filename)
+    df_csv = df.to_csv(index=False)
+    blob.upload_from_file(BytesIO(df_csv.encode('utf-8')), content_type='text/csv')
+
+class ParticipantSummaryDataDump(ToolBase):
+    logger_name = None
+
+    def run(self):
+        summary_list = []
+        super(ParticipantSummaryDataDump, self).run()
+        with self.get_session() as session:
+            summary_dao = ParticipantSummaryDao()
+            # --id option takes precedence over --from-file option
+            if self.args.id:
+                participant_id_list = [int(i) for i in self.args.id.split(',')]
+            elif self.args.from_file:
+                participant_id_list = self.get_int_ids_from_file(self.args.from_file)
+            else:
+                # Default to all participant_summary ids
+                participant_id_list = session.query(
+                    ParticipantSummary.participantId,
+                ).filter(
+                    Participant.isTestParticipant == 1
+                ).order_by(ParticipantSummary.participantId).all()
+
+            count = 0
+            last_id = None
+            total_rows = len(participant_id_list)
+            chunk_size = 10000
+            for id_list_subset in list_chunks(lst=participant_id_list, chunk_size=chunk_size):
+                logging.info(f'{datetime.now()}: {count} of {len(participant_id_list)} (last id: {last_id})')
+
+                summary_list = session.query(
+                    ParticipantSummary
+                ).filter(
+                    ParticipantSummary.participantId.in_(id_list_subset),
+                    Participant.isTestParticipant == 1
+                ).all()
+                results = [summary_dao.to_client_json(result) for result in summary_list]
+                ignored_columns = [
+                    'biobankId',
+                    'firstName',
+                    'middleName',
+                    'lastName',
+                    'zipCode',
+                    'state',
+                    'city',
+                    'streetAddress',
+                    'streetAddress2',
+                    'phoneNumber',
+                    'loginPhoneNumber',
+                    'email',
+                    'dateOfBirth'
+                ]
+                for column in results:
+                    for i in ignored_columns:
+                        if i in column:
+                            del column[i]
```

Comment:
The name `column` here is really confusing. `column` is each item of the `results` list, so it's really the dictionary of each participant's data, right? Would participant or participant_dict potentially be a more meaningful name here?